### PR TITLE
Upgrade dependency before rollout 3.0.0

### PR DESCRIPTION
**Pull request by andyduong1920**
## What happened

Upgrade dependency before rollout 3.0.0
 
## Proof Of Work

The test is passed

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -31,11 +31,11 @@ defmodule NimbleTemplate.MixProject do
...
-      {:jason, "~> 1.2.2"},
-      {:httpoison, "~> 1.7.0"},
-      {:mimic, "~> 1.3.1", only: :test},
-      {:credo, "~> 1.4.0", only: [:dev, :test], runtime: false},
-      {:ex_doc, "~> 0.23.0", only: :dev, runtime: false}
+      {:jason, "~> 1.2"},
+      {:httpoison, "~> 1.7"},
+      {:mimic, "~> 1.3", only: :test},
+      {:credo, "~> 1.4", only: [:dev, :test], runtime: false},
+      {:ex_doc, "~> 0.23", only: :dev, runtime: false}
```
Comment (byhbt) [inline mix.exs:38]: hi @andyduong1920 !

Should we arrange these dependencies by alphabet?

Comment (andyduong1920) [reply mix.exs:38]: Sure @byhbt :D

Comment (byhbt) [reply mix.exs]: btw, i see this issue happened many times, should we make it as convention? :)

Comment (andyduong1920) [reply mix.exs]: Agree, we should add it to our convention @byhbt 👍

Comment (andyduong1920) [reply mix.exs]: Fixed it in https://github.com/nimblehq/elixir-templates/pull/97/commits/f99c162050128c1356275faa603ffa0906746c4a

Comment (andyduong1920) [reply mix.exs]: Could you open an issue on our compass to adding this issue to our Elixir convention @byhbt 🤔

Comment (byhbt) [reply mix.exs]: Sure thing, I will do that today. Thanks

Comment (andyduong1920) [reply mix.exs]: Thanks @byhbt

**Review by byhbt — APPROVED** [review]

**Review by bterone — APPROVED** [review]

**Pull request merged**